app/views/local_authorities/show.html.erb

```diff
@@ -13,7 +13,7 @@
 
```

Comment:
As this change is frontend it would be useful to have an accompanying screenshot. Would it be possible to add one? I ran the application and the table looks good, and is working as expected. Nice work in thinking of this, as it's really useful!